runs/run_eigendec.py

```diff
@@ -39,9 +39,6 @@
 from fenics_ice.config import ConfigParser
 from fenics_ice.decorators import count_calls, timer, flagged_error
 
-import slepc4py.SLEPc as SLEPc
-import petsc4py.PETSc as PETSc
```

Comment:
@jrmaddison I assume PETSc is not longer needed?

Reply:
No, it's unused. I've updated the summary.

Reply:
Thanks I will merge this then and the other one as well.